test.py

```diff
@@ -0,0 +1,4 @@
+
+from instructor.compiler import *
+
+print preview_as_json(1L)
```

Comment:
Yeah, remove this please.  And I made it to the bottom of my first read through!!!! Yay!